samples/msal-angular-v3-samples/angular17-standalone-sample/README.md

```diff
@@ -21,7 +22,7 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 ## Local development
 
-If you are trying to run this sample locally in the MSAL.js repo, run `npm run build` before `npm run start` to install a tarball file of MSAL Angular.
+If you are trying to run this sample locally in the MSAL.js repo, run `npm run build` before `npm run start` to install a tarball file of MSAL Angular. You may also need to remove `"e2e-test-utils": "^0.0.1",` from the package.json for this to work.
```

Comment:
Instead can we just change the dependency to reference the folder rather than version? This is failing because folks aren't installing from the root of the repo prior to attempting install of this package (workspaces don't get initialized).

Reply:
@jo-arroyo it's your call :) If am fine with both options

Reply:
Sure, that's fine. @konstantin-msft, feel free to remove this change from this PR, I will change all the samples in another one. Thanks!